.ci/Jenkinsfile

```diff
@@ -25,6 +25,7 @@ pipeline {
   }
   parameters {
     choice(name: 'runTestsSuite', choices: ['all', 'helm', 'ingest-manager', 'metricbeat'], description: 'Choose which test suite to run (default: all)')
+    booleanParam(name: "SKIP_GIT_CHECKS", defaultValue: "false", description: "If it's needed to check for Git changes to filter by modified sources")
```

Comment:
IIRC, we use the `force` parameter in some other cases, what do you think to rename it?

Reply:
I like the idea, being consistent across projects will bring maintainability!

Reply:
I'm still thinking about this: what do we want to `force` here? When triggering the build, we want to force not checking for modified paths, so not sure if `force` is providing context enough to understand what is going on.


I'd prefer using `forceSkipGitChecks` instead, as it clearly indicates what will happen. Keeping the `force` as a prefix brings consistency, although not sure if the variable name is still too long.

wdyt?

Reply:
I'd say something generic and atomic could help with, but we are not there yet, there are certain cases where we use "run_as_the_master_branch". So no strong opinion in my end

Reply:
Mmmm, interesting. That's making me think that the `master` branch for this project does not get special behavior, and maybe it should.

Anyway, thanks to this healthy discussion, I'll use `forceSkipGitChecks` to be consistent with the `force` parameter. We can rethink the behavior for master branch later on.

Thanks!